### PR TITLE
Nuxt-pinia-tailwind create gist panel

### PR DESCRIPTION
**Pull request by markshenouda**
Screen Shot 2022-09-28 at 8 03 32 PM

### CODE REVIEW

**Review by hdJerry — APPROVED** [review]
LGTM

File: nuxt-pinia-tailwind/components/UserGists/UserGists.vue [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
 <script lang="ts">
-export default {};
+import Vue from 'vue';
+import { useUserStore } from '~/store/userStore';
+
+export default Vue.extend({
+  name: 'UserGists',
+  setup() {
+    const userStore = useUserStore();
+    userStore.getUserGists();
```
Comment (jesus4497) [inline nuxt-pinia-tailwind/components/UserGists/UserGists.vue:21]: We should do this fetch in the server side

```suggestion
   const userStore = useUserStore();

   useAsync(async () => {
      await userStore.getUserGists();
    });
```

File: nuxt-pinia-tailwind/nuxt.config.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ export default {
     // https://go.nuxtjs.dev/axios
     '@nuxtjs/axios',
     '@nuxtjs/auth-next',
+    '@pinia/nuxt',
```
Comment (jesus4497) [inline nuxt-pinia-tailwind/nuxt.config.js:50]: Good catch!

**Review by jesus4497 — CHANGES_REQUESTED** [review]
LGTM! Some small changes. 

Although we need to add unit tests

File: next-react-query-tailwind/public/mockServiceWorker.js [hunk 3]
```diff
@@ -8,329 +8,331 @@
...
+  })
+}
+
+function uuidv4() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+    const r = (Math.random() * 16) | 0
+    const v = c == 'x' ? r : (r & 0x3) | 0x8
+    return v.toString(16)
+  })
+}
```
[the remark this answers is not in the document]
Comment (hdJerry) [reply next-react-query-tailwind/public/mockServiceWorker.js:338]: Probably prettier changed it.

**Review by jesus4497 — APPROVED** [review]

**Pull request merged**